backoffice/pages/catalog/brands/[id]/edit.tsx

```diff
@@ -62,31 +62,35 @@ const BrandEdit: NextPage = () => {
   }, [id]);
 
   if (isLoading) return <p>Loading...</p>;
-  return (
-    <div className="row mt-5">
-      <div className="col-md-8">
-        <h2>Edit brand: {id}</h2>
-        <form onSubmit={handleSubmit(handleSubmitEdit)}>
-          <BrandGeneralInformation
-            register={register}
-            errors={errors}
-            setValue={setValue}
-            trigger={trigger}
-            brand={brand}
-          />
+  if (!brand) {
+    return <p>No brand</p>;
```

Comment:
UI rendering should be handled by BrandGeneralInformation instead of create.tsx/edit.tsx
Incase of editing we always have brand so no need to check this

Reply:
Hi @khanhtrand , I tried to remove this line but the function worked incorrectly. 
Because the input field becomes an uncontrolled component and managed by React Hook Form. First time, the default value of the input field is undefined.  After re render, the default value still remains if don't check this.
This line is to ensure that the value passed to the React Hook Form is always deterministic. Can you review this MR again?